fava_investor/modules/performance/accounts_config.py

```diff
@@ -0,0 +1,41 @@
+import re
+from typing import List
+
+from core import FavaLedger
+
+from fava_investor.modules.performance.balances import ConfigDict
+
+
+class AccountsConfig:
```

Comment:
We can fully get rid of accounts_config.py. See my other comments for this. Instead, see my other modules (eg: tlh or cashdrag). They use accapi (favainvestorapi), and the config.